app/navigation/Navigation.scala

```diff
@@ -168,4 +170,14 @@ class Navigation @Inject()(dateTimeHelper: DateTimeHelper,
         routes.OtherReasonController.onPageLoadForUploadComplete(mode)
     }
   }
+
+  def routeForUploadEvidenceQuestion(isUploadEvidence: Option[String], request: UserRequest[_], mode: Mode) :Call = {
+    isUploadEvidence match {
+      case Some(ans) if ans.equalsIgnoreCase("yes") => routes.OtherReasonController.onPageLoadForUploadEvidence(mode)
+      case Some(ans) if ans.equalsIgnoreCase("no") => routes.CheckYourAnswersController.onPageLoad()
```

Comment:
You would have to route to either the making a late appeal page or the CYA page, there is a method to help: 
```suggestion
      case Some(ans) if ans.equalsIgnoreCase("no") => routeToMakingALateAppealOrCYAPage(request, mode)
```